source/_components/sensor.nest.markdown

```diff
@@ -38,6 +38,8 @@ Configuration variables:
 
 The following conditions are available by device:
 
+- Nest Home:
+  - eta: It is readonly currently, change service will be provided in the future version.
```

Comment:
Please describe "eta". It's an estimate of ...? May be provide an example value. The notice about features in future can be removed IMO.